client/etcd/options.go

```diff
@@ -30,13 +30,20 @@ import (
 
 	"github.com/m3db/m3cluster/services"
 	"github.com/m3db/m3x/instrument"
+	"github.com/m3db/m3x/retry"
 )
 
 const (
 	defaultKeepAliveEnabled         = false
 	defaultKeepAlivePeriod          = 30 * time.Minute
 	defaultKeepAlivePeriodMaxJitter = 0
 	defaultKeepAliveTimeout         = 20 * time.Second
+
+	defaultRetryInitialBackoff = 2 * time.Second
+	defaultRetryBackoffFactor  = 2.0
+	defaultRetryMaxBackoff     = 4
```

Comment:
Should this be `4 * time.Second` rather?

Reply:
+1

Reply:
Sure thing, I'll just set this to zero.